did_doc_sov/src/service/aip1.rs

```diff
@@ -59,6 +62,15 @@ impl TryFrom<Service<ExtraFieldsSov>> for ServiceAIP1 {
     }
 }
 
+impl TryFrom<Service<HashMap<String, Value>>> for ServiceAIP1 {
+    type Error = DidDocumentSovError;
+
+    fn try_from(service: Service<HashMap<String, Value>>) -> Result<Self, Self::Error> {
+        let extra = serde_json::from_value::<ExtraFieldsAIP1>(serde_json::to_value(service.extra())?)?;
```

Comment:
looks like will never fail currently, since `ExtraFieldsAIP1 {}` doesn't have any fields

Reply:
Added `deny_unknown_fields` (seems to work somehow).